<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>360</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Dummy Files Creator</string>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="widget_window">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>530</width>
     <height>350</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>530</width>
     <height>350</height>
    </size>
   </property>
   <widget class="QWidget" name="widget_main" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>510</width>
      <height>320</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>510</width>
      <height>320</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>510</width>
      <height>320</height>
     </size>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>514</width>
       <height>318</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layout_main" rowstretch="0,0,0,0,0,0,0,0" columnstretch="0,0,0">
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>2</number>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="text_n_files">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="text_path">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_path">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Save to Folder</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <layout class="QHBoxLayout" name="layout_verbose_chunk">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontal_spacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>30</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkbox_verbose">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Affects performance&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="text">
           <string>Verbose</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontal_spacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_chunk_size">
          <property name="minimumSize">
           <size>
            <width>75</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>Chunk Size</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="text_chunk_size">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>23</height>
           </size>
          </property>
          <property name="text">
           <string>1024</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_chunk_unit">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>23</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>KiB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MiB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GiB</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="horizontal_spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="checkbox_savelog">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>23</height>
           </size>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Affects performance&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Save log file</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontal_spacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="text_size_files">
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>23</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="button_browse_files">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="layout_file_unit">
        <item>
         <widget class="QComboBox" name="combo_file_unit">
          <property name="minimumSize">
           <size>
            <width>65</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>65</width>
            <height>23</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <property name="frame">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>KiB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MiB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GiB</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_size_files">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Size of Files</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QWidget" name="widget_buttons" native="true">
        <property name="minimumSize">
         <size>
          <width>510</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>510</width>
          <height>30</height>
         </size>
        </property>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>512</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="layout_button">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="button_create_stop">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_close_quit">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>25</height>
             </size>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_n_files">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Number of Files</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QWidget" name="widget_progress_bar" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>510</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>510</width>
          <height>50</height>
         </size>
        </property>
        <widget class="QProgressBar" name="progress_bar">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>505</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>505</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>505</width>
           <height>20</height>
          </size>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="label_progress">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>510</width>
           <height>20</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>510</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>0/0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <layout class="QHBoxLayout" name="layout_log_external">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_log" native="true">
          <property name="minimumSize">
           <size>
            <width>510</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>510</width>
            <height>50</height>
           </size>
          </property>
          <widget class="QWidget" name="layoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>511</width>
             <height>54</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
            <item row="0" column="1">
             <widget class="QLineEdit" name="text_logfilepath">
              <property name="minimumSize">
               <size>
                <width>300</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>300</width>
                <height>23</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
             <widget class="QCheckBox" name="checkbox_md5hash">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>120</width>
                <height>21</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>120</width>
                <height>21</height>
               </size>
              </property>
              <property name="toolTip">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Affects performance&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="text">
               <string>Save md5 hash</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="button_browse_log">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Browse</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_logfilepath">
              <property name="minimumSize">
               <size>
                <width>85</width>
                <height>23</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>85</width>
                <height>23</height>
               </size>
              </property>
              <property name="text">
               <string>Log to Folder</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item row="7" column="0" colspan="3">
       <layout class="QHBoxLayout" name="layout_verbose">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="widget_verbose" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>510</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>510</width>
            <height>50</height>
           </size>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>512</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="layout_verbose_vert">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_verbose_information">
              <property name="minimumSize">
               <size>
                <width>510</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>510</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>Verbose progress bar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="progress_bar_verbose">
              <property name="minimumSize">
               <size>
                <width>505</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>505</width>
                <height>20</height>
               </size>
              </property>
              <property name="value">
               <number>0</number>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>530</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>text_path</tabstop>
  <tabstop>button_browse_files</tabstop>
  <tabstop>text_n_files</tabstop>
  <tabstop>text_size_files</tabstop>
  <tabstop>combo_file_unit</tabstop>
  <tabstop>checkbox_verbose</tabstop>
  <tabstop>text_chunk_size</tabstop>
  <tabstop>combo_chunk_unit</tabstop>
  <tabstop>checkbox_savelog</tabstop>
  <tabstop>text_logfilepath</tabstop>
  <tabstop>button_browse_log</tabstop>
  <tabstop>checkbox_md5hash</tabstop>
  <tabstop>button_create_stop</tabstop>
  <tabstop>button_close_quit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
